### PR TITLE
A first pass on the 'requires attention' section WIP

### PR DESCRIPTION
**Pull request by jesseyuen**
This PR:
- adds an early concept for how might might show alerts that require our users attention. I expect this design to change. I mean CIRCLES! 
- adds a feature flag in the /admin section

We'll need @gregknight1 to input from a content perspective.

homapge

### CODE REVIEW

File: app/views/home.html [hunk 1]
```diff
@@ -42,6 +45,30 @@ <h2 class="govuk-heading-m"><a href="/data-requirements" class="govuk-link govuk
 
     <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
+    {% if data.settings.showRequiresAttentionSection %}
```
Comment (edwardhorsford) [inline app/views/home.html]: This should also check if the counts of either of your two variables are greater than 0

File: app/views/home.html [hunk 2]
```diff
@@ -42,6 +45,30 @@ <h2 class="govuk-heading-m"><a href="/data-requirements" class="govuk-link govuk
...
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+          <h2 class="govuk-heading-m">Requires your attention</h2>
+          <div class="app-attention-link">
+            <span class="app-attention-link__count">{{applyDraftCount}}</span>
+            <div class="app-attention-link__content">
+              <a href="/records?filterStatus=Draft" class="govuk-link app-attention-link__text govuk-!-font-size-24">Apply drafts are ready to review</a>
```
Comment (edwardhorsford) [inline app/views/home.html]: Do you need the manual font size here? could `app-attention-link__text` set the size?

**Review by edwardhorsford — COMMENTED** [review]
Looking good man!

I'm not sure about the name `attention-link`.

File: app/views/home.html (hunk 1 above)
Comment (jesseyuen) [reply app/views/home.html]: And I imagine also a check within to only show each link when the count is > 0

**Review by edwardhorsford — APPROVED** [review]

**Comment by gregknight1** [comment]
Sorry I'm late to this @jesseyuen! Few observations and questions:

- Think 'attention' is a bit vague - maybe the heading could be something like 'Action required' as they need to do something. Do we say anything similar anywhere else in the service?
- Apply drafts link could be: (7) Apply drafts ready for review
- cycle end link: (21) Records for review before cycle ends

Happy to chat about any of this. Looking good.

**Comment by jesseyuen** [comment]
> Sorry I'm late to this @jesseyuen! Few observations and questions:
> 
> * Think 'attention' is a bit vague - maybe the heading could be something like 'Action required' as they need to do something. Do we say anything similar anywhere else in the service?
> * Apply drafts link could be: (7) Apply drafts ready for review
> * cycle end link: (21) Records for review before cycle ends
> 
> Happy to chat about any of this. Looking good.

Thanks @gregknight1. Will add those changes in.

**Pull request merged**